diagnosis_api/controllers/search_disease_by_syptoms.py

```diff
@@ -32,37 +32,44 @@ def search_disease_by_symptoms(self):
         """
 
         # check if the file exists
-        basepath = 'diagnosis_api/dataset/'
+        basepath = "diagnosis_api/dataset/"
         files = []
         for entry in os.listdir(basepath):
             if os.path.isfile(os.path.join(basepath, entry)):
                 files.append(entry)
 
         if self.file_name.lower() not in files:
-            return ({'message': 'File with that name doesnot exist'})
+            return {"message": "File with that name doesnot exist"}
 
         # check for non excel files
-        if (not self.file_name.lower().endswith(('.xls', '.xlsx'))):
-            return ({'message': 'Dataset file format not supported, only excel files are accepted'})
+        if not self.file_name.lower().endswith((".xls", ".xlsx")):
+            return {
+                "message": "Dataset file format not supported, only excel files are accepted"}
 
         # load disease file
-        diseases = pd.read_excel('diagnosis_api/dataset/{}'.format(self.file_name))
+        diseases = pd.read_excel(
```

Comment:
You can create a unction for reading from excel file otherwise you are going to repeat this implementation everytime you need to read from an excel file

Reply:
Very true about this. i am going to fix it